index.html

```diff
@@ -3250,38 +2503,31 @@ <h4>Additional information</h4>
 					<p>Run automated tasks only when necessary to reduce unnecessary resource utilisation.</p>
 				</section>
-				<section class="notoc" data-testable="human" data-materials="low" data-energy="low" data-water="low" data-emissions="low" data-standard="AWS WAF, Azure WAF, GR491" data-considerations="Security" data-categories="Performance, Software">
+				<section class="notoc" data-materials="low" data-energy="low" data-water="low" data-emissions="low" data-standard="AWS WAF, Azure WAF, GR491" data-considerations="Security" data-categories="Performance, Software">
 					<h4 id="automated-scaling">Success Criterion: Automated scaling</h4>
-					<p class="external"><span class="human">Human-testable</span> <span class="hide">and </span> <a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#HIS06-3">Resources</a></p>
-					<p>Use automated scaling to promptly adjust server capacity up or down based on demand, ensuring efficient resource allocation. Implement buffering and throttling to manage load and maintain performance without overprovisioning.</p>
+					<p class="external"><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#HIS06-3">Resources</a></p>
+					<p>Use automated scaling to adjust server capacity based on demand, ensuring efficient resource allocation during traffic spikes. Implement buffering and throttling to manage load and maintain performance without overprovisioning. Also use automation to promptly scale resources back down based on demand.</p>
 				</section>
-				<section class="notoc" data-testable="machine" data-materials="low" data-energy="low" data-water="low" data-emissions="low" data-standard="" data-considerations="Security" data-categories="AI, Performance, Software">
+				<section class="notoc" data-materials="low" data-energy="low" data-water="low" data-emissions="low" data-standard="" data-considerations="Security" data-categories="AI, Performance, Software">
 					<h4 id="suspicious-activity-filtering">Success Criterion: Suspicious activity filtering</h4>
-					<p class="external"><a class="machine" href="https://w3c.github.io/sustainableweb-wsg/star.html#HIS06-4">Machine-testable</a> <span class="hide">and </span><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#HIS06-4">Resources</a></p>
+					<p class="external"><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#HIS06-4">Resources</a></p>
 					<p>Restrict the activity of unwanted and unnecessary third-party crawlers, suspicious user agents, unwanted users, bots, and scrapers from accessing or downloading your content. Follow best practices, such as server access rules and security tools, while ensuring your content remains accessible to users, search engines and any helpful, welcome crawlers. Consider that scrapers may be used to inform and train large language models.</p>
```

Comment:
proposed revision is:
Discourage unwanted and unnecessary third-party crawlers from accessing or downloading your content by using best practices such as robots.txt while ensuring your content remains accessible to search engines and any helpful, welcome crawlers. (@susannah-hill @ryansholin )